packages/contentstack-export-to-csv/src/util/index.js

```diff
@@ -396,12 +396,34 @@ function getOrgUsers(managementAPIClient, orgUid, ecsv) {
         if (!organization.getInvitations) {
           return reject(new Error(config.adminError));
         }
-        organization.getInvitations().then((users) => resolve(users));
+        try {
+          const users = await getUsers(organization, { skip: 0, page: 1, limit: 100 });
+          return resolve({ items: users });
+        } catch (error) {
+          return reject(error);
+        }
       })
       .catch((error) => reject(error));
   });
 }
 
+async function getUsers(organization, params, result = []) {
+  try {
+    const users = await organization.getInvitations(params);
+    if (!users.items || (users.items && !users.items.length)) {
+      return result;
+    } else {
+      result = result.concat(users.items);
+      params.skip = params.page * params.limit;
+      params.page++;
+      await wait(200);
```

Comment:
Is there any specific reason why added this delay ?, If so we need to consider having a configuration where we keep the delay.

Reply:
@shafeeqd959 the following was the @vkalta answer when it was asked earlier. `This is intended to not overload the server with requests.`

@vkalta take please it forward for any further queries.

CC @netrajpatel